frontend/.eslintrc.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: [
+    "plugin:react/recommended",
+    "standard"
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 12,
+    sourceType: "module"
+  },
+  plugins: [
+    "react",
+    "@typescript-eslint"
+  ],
+  rules: {
+    quotes: ["error", "double"],
+    // Keeping no-unused-vars on had some weird behavior with enums
+    semi: ["error", "always"],
+    "no-unused-vars": "off",
+    // note you must disable the base rule as it can report incorrect errors
+    "no-use-before-define": "off",
+    "react/react-in-jsx-scope": "off",
+    // allow jsx syntax in js files (for next.js project)
+    "react/jsx-filename-extension": [1, { extensions: [".js", ".jsx", ".ts", ".tsx"] }] // should add ".ts" if typescript project
```

Comment:
Who said anything about next.js? The advantage of keeping jsx to jsx/tsx is that .ts can use <> syntax, plus, that way the linter will ensure our file extensions are as-expected.